python/ray/train/BUILD

```diff
@@ -179,7 +179,7 @@ py_test(
 
 py_test(
     name = "test_huggingface_predictor",
-    size = "medium",
+    size = "large",
```

Comment:
I separated `test_huggingface_checkpoint` into its own test file and reverted this back to medium. The test was timing out originally since there were so many tests in that file.